### ISSUE TITLE
The reconstruction image degrade to the gray

### ISSUE THREAD

**Issue by tianjiu233** [Zian-Xu/Swin-MAE#4]
Hi,
I have tried your code on a image dataset.
However, after the training about 40 epochs, I tried the swin_mae on the image I used, 
image reconstructed by the model is always the gray one.
Have you met the case and might it be solved?
Hope you help and thanks for your time.

**Comment by Zian-Xu** [comment]
According to your description, for some reason, the trained model cannot reconstruct the original image. I suggest checking the training loss curve. In the experiments I have done before, after 400 training epochs, the final loss of the model was around 0.0025. If your loss is too large, the model is indeed performing poorly. The possible reason is insufficient training epoch. If your loss is normal, then you should troubleshoot whether the model is being used in the right way. I look forward to your further detailed description of the problem.
